test/perf-test/cleanup.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
why not make the `cleanup.sh` a function of the perf.sh ?

Reply:
Makes sense,
I would recommend that you add logic to both the `cleanup.sh` script and `perf.sh` script to be able to run them without having to run it from the directory.

Reply:
Done